path lookup: wordsmithing (#195)
scionassociation/scion-cp_I-D: draft-dekater-scion-controlplane.md

2. The Control Service of the source AS answers each request with  a `SegmentsResponse` message. Specifically, for each segment type:

>    - up segments are stored in the path database of the local Control Service and can be returned immediately.

Reviewer: ```suggestion
   - up segments: The local Control Service has up segments stored in the path database and returns them immediately.
```

Reviewer: Change to be in line with core/down

>    - core segments may be cached. Otherwise, the local Control Service queries the Control Services of the reachable core ASes in the source ISD for core segments to core ASes in the destination ISD. To reach the core Control Services, the Control Service of the source AS uses the locally stored up segments. Once obtained, it returns the core segments.

Author: ```suggestion
   - core segments: The local Control Service may return locally cached core segments or query the Control Services of the reachable core ASes in the source ISD. These core ASes return core segments to core ASes in the destination ISD. To reach the core Control Services, the Control Service of the source AS uses the locally stored up segments. Once obtained, it returns the core segments and adds them to the local cache for future request.
```
